<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="comboBoxA">
        <item>
         <property name="text">
          <string>Persion</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextEditA"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="translateButton">
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>=&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QComboBox" name="comboBoxB">
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Persion</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="TextEditB"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="separator"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuTranslator">
    <property name="title">
     <string>Translator</string>
    </property>
    <addaction name="action_Spell_Check"/>
    <addaction name="action_Translate"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
    <addaction name="separator"/>
    <addaction name="action_Add_new_word"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTranslator"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>new</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionDictionaries">
   <property name="text">
    <string>&amp;Translate</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
  </action>
  <action name="action_Dictionaries">
   <property name="text">
    <string>&amp;Dictionaries</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Translate">
   <property name="text">
    <string>&amp;Translate</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="text">
    <string>&amp;Options</string>
   </property>
  </action>
  <action name="action_Spell_Check">
   <property name="text">
    <string>&amp;Spell Check</string>
   </property>
  </action>
  <action name="action_Add_new_word">
   <property name="text">
    <string>&amp;Add new word</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
